src/angular-maps/esri-web-scene/esri-web-scene/esri-web-scene.component.ts

```diff
@@ -117,24 +142,29 @@ export class SbbEsriWebScene implements OnInit {
     this.sceneView.watch('camera', (camera) => this._handleCameraChange(camera));
   }
 
-  private _handleMouseClick(e: __esri.SceneViewClickEvent) {
+  private _handleMouseClick(e: SceneViewClickEvent) {
     this._hitTestService
       .esriHitTest(this.sceneView, e)
       .then((hitTestGraphics) =>
         this.mapClick.emit({ clickedPoint: e.mapPoint, clickedGraphics: hitTestGraphics })
       );
   }
 
-  private _handleCameraChange(camera: __esri.Camera) {
+  private _handleCameraChange(camera: Camera) {
     this.cameraChanged.emit(camera);
   }
 
-  private _createNewCamera(newCam: SbbEsri3DCamera): __esri.Camera {
-    return new this._esri.Camera({
-      position: newCam.position,
-      tilt: newCam.tilt,
-      heading: newCam.heading,
-      fov: newCam.fov,
-    });
+  private _destroySceneView() {
+    // it was in 4.18, but just to be sure it's cleaned-up: https://community.esri.com/t5/arcgis-api-for-javascript/4-17-memory-issue-angular/td-p/140389
+    const sceneView = this.sceneView;
+    const map = this.webScene;
+    if (sceneView) {
+      sceneView?.destroy();
+    }
+
+    if (map) {
+      map.removeAll();
+      map.destroy();
+    }
```

Comment:
I believe this could be shortened to
```suggestion
    this.sceneView?.destroy();
    this.webScene?.removeAll();
    this.webScene?.destroy();
```